packages/phone-number-privacy/src/salt-generation/bls-cryptography-client.ts

```diff
@@ -0,0 +1,35 @@
+import { AzureKeyVaultClient } from '@celo/contractkit/src/utils/azure-key-vault-client'
+import { BLINDBLS } from 'bls12377js-blind'
+import config from '../config'
+
+export class BLSCryptographyClient {
+  /*
+   * Computes the BLS Salt for the blinded phone number.
+   */
+  public static async computeBLSSalt(queryPhoneNumber: string) {
+    try {
+      const privateKey = await BLSCryptographyClient.getPrivateKey()
+      const pkBuff = new Buffer(privateKey)
+      return BLINDBLS.computePRF(pkBuff, new Buffer(queryPhoneNumber))
+    } catch (e) {
+      console.error('Failed to compute salt', e)
+      throw e
+    }
+  }
+
+  private static privateKey: string
```

Comment:
does this work to have this static variable named the same as the const variable in the above method?

Reply:
Yeah, the method above can distinguish between them `privateKey` is the local var and `BLSCryptographyClient.privateKey` is the static one.

Reply:
👍